utilities/decrypt_data/decrypt_object.py

```
@@ -0,0 +1,112 @@
+            json=[payload],            # API expects a list of dicts
+            headers={"Content-Type": "application/json"},
+            timeout=10
+        )
+        resp.raise_for_status()
+        result = resp.json()           # e.g. [ {key1: plain1}, {key2: plain2} ]
+        out = {}
+        for entry in result:
+            out.update(entry)
+        return out
+
+    except Exception as e:
```

Ruff (0.11.9)

Do not catch blind exception: `Exception`

(BLE001)